packages/flutter_tools/lib/src/compute_dev_dependencies.dart

```diff
@@ -2,126 +2,136 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'base/logger.dart';
-import 'convert.dart';
-import 'dart/pub.dart';
+import 'package:file/file.dart';
```

Comment:
I don't think we should be importing `package:file` here. Instead, we should be importing `base/file_system.dart`.

Reply:
Thanks!